packages/scandipwa/src/component/Header/Header.component.js

```diff
@@ -81,6 +81,7 @@ export class Header extends NavigationAbstract {
             onBackClick: PropTypes.func,
             title: PropTypes.string
         }).isRequired,
+        productsInWishlist: PropTypes.objectOf(PropTypes.object),
```

Comment:
Object is deprecated proptype. Check proptypes in Types same as check where same state is get, maybe you can just reuse already made logic.